helm/operator/templates/minio.min.io_tenants.yaml

```diff
@@ -78,6 +78,8 @@ spec:
                     type: string
                 type: object
                 x-kubernetes-map-type: atomic
+              enableSFTP:
```

Comment:
Do we need a change in `kubectl-minio` as well to support `--sftp` args? something like `--enable-sftp` and `--sftp-port`

Reply:
Also add a sample commented entry in `examples/kustomization/base/tenant.yaml`

Reply:
@shtripat thanks, done.